src/renderer/components/Title.tsx

```diff
@@ -1,48 +1,17 @@
 import * as React from 'react';
 
+//import helpers
+import { getStatic } from '../helpers/static';
+
 type Props = {
   // projectName: string;
 };
 
 const Title: React.FC<Props> = props => (
-  <>
```

Comment:
Why do we want to replace the svg? It just seems more flexible than an img tag.

Reply:
In my opinion, image tags are simpler, the classes that the svg were using are from adobe illlustrator and not really meant to be accessed in this way. In addition, we're storing the logo image within the static folder so I feel like we should use it and also maintenance to the logo becomes easier because all we need to do is swap out image files for future changes